src/firefly/js/voAnalyzer/vo-typedefs.jsdoc

```diff
@@ -154,6 +157,8 @@
  * @prop {CISXui} [cisxUI] - names should be one of: HiPS, FOV, hips_initial_ra, hips_initial_dec, moc, examples, hipsCtype1, hipsCtype2
  * @prop {Array.<ServiceDescriptorInputParam>} [cisxTokenSub]
  * @prop {Array.<ServiceDescriptorInputParam>} serDefParams
+ * @prop {WorldPt} positionWP if there is a MetaConst.SEARCH_TARGET in datalink table it is propagated here
+ * @prop {WorldPt} rowWP if there is a MetaConst.ROW_TARGET in datalink table it is propagated here
```

Comment:
`serDef` was used throughout the code, but its purpose wasn’t immediately clear to me. I initially thought it referred to a default serialization mode, which does not make sense. Renaming it to better indicate that it represents a map rather than a single value would make its purpose clearer.  Maybe `svcDescInputs` or `svcDescParams`.